src/include/uapi/abi.h

```diff
@@ -52,7 +52,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 1
+#define SOF_ABI_MINOR 2
```

Comment:
This will introduce a compatibility break with EQs due to process/type IPC changed so the major version should be bumped. Though it's not yet an official released feature (=not validated) so this might be OK but I will need to fix other things after this. I've asked opinion from @lgirdwood and @plbossart for version bump need.